Docs/source/install/hpc/perlmutter.rst

```diff
@@ -288,5 +288,4 @@ For post-processing, most users use Python via NERSC's `Jupyter service <https:/
 Please follow the same process as for :ref:`NERSC Cori post-processing <post-processing-cori>`.
 **Important:** The *environment + Jupyter kernel* must separate from the one you create for Cori.
 
-The Perlmutter ``$PSCRATCH`` filesystem is currently not yet available on Jupyter.
-Thus, store or copy your data to Cori's ``$SCRATCH`` or use the Community FileSystem (CFS) for now.
+The Perlmutter ``$PSCRATCH`` filesystem as Cori's ``$SCRATCH`` are available on Jupyter. You can also use the Community FileSystem (CFS) from everywhere.
```

Comment:
Thank you for the update!

How about the following slight update to clarify a bit more?
```suggestion
The Perlmutter ``$PSCRATCH`` filesystem is only available on *Perlmutter* Jupyter nodes.
Likewise, Cori's ``$SCRATCH`` filesystem is only available on *Cori* Jupyter nodes.
You can use the Community FileSystem (CFS) from everywhere.
```

Reply:
Yes, absolutely, it's clearer!